app/views/admin/especialidades/_form_new.html.erb

```diff
@@ -1,19 +1,19 @@
 <div class="container down">
   <!-- Modal HTML Markup -->
   <div id="especialidad-modal" class="modal fade">
-  <div class="modal-dialog" role="document">
-    <div class="modal-content">
+    <div class="modal-dialog" role="document">
```

Comment:
El indentado en este documento se encuentra correcto, por alun motivo aca muestra espacios en la linea 4 y 5 que en realidad no existen, adjunto imagen de prueba de ello.

![in](https://user-images.githubusercontent.com/54487801/77868251-21271f80-7200-11ea-9a2a-df02b3917173.png)